api/Node.json

```diff
@@ -882,10 +882,10 @@
               "version_added": true
             },
             "safari": {
-              "version_added": true
+              "version_added": "≤4"
```

Comment:
No suggestion here, no `isDefaultNamespace` in https://trac.webkit.org/browser/webkit/trunk?rev=4552